### PR TITLE
Updated the client to reflect changes in httplug

### PR DESCRIPTION
**Pull request by Nyholm**
I know we put other adapters than guzzle6 on hold until these libraries are stable. But I needed this adapter to properly test my application.

### CODE REVIEW

**Comment by sagikazarmark** [comment]
On hold means we don't want to develop them until the contract is stable, but we are open to contributions. I need to ask some patience as I am busy with other things in the next few days. If someone else reviews it, it can be merged, otherwise it will be on hold at least till the weekend.

**Comment by Nyholm** [comment]
Yeah of course. No hurry. I got some tests to fix =)

**Comment by Nyholm** [comment]
This PR is ~~not~~ NOW ready for review

**Comment by sagikazarmark** [comment]
Not?

**Comment by Nyholm** [comment]
Sorry, It's late. "NOW"

=)

File: .travis.yml [hunk 1]
```diff
@@ -16,15 +26,19 @@ matrix:
       env:
         - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
         - COVERAGE=true
-        - PHPUNIT_FLAGS="--coverage-clover build/coverage.xml"
+        - TEST_COMMAND="composer test-ci"
```
Comment (sagikazarmark) [inline .travis.yml]: Please check this again:

https://github.com/php-http/httplug/blob/master/.travis.yml#L26

Travis linter says multiline event params are invalid here

**Comment by sagikazarmark** [comment]
:+1: but it's late, so I would wait for another review (even if it is me tommorrow with clear mind :smile: )

**Comment by Nyholm** [comment]
Thank you for taking the time.

**Comment by joelwurtz** [comment]
LGTM

**Comment by sagikazarmark** [comment]
Agree, thanks @Nyholm

**Comment by Nyholm** [comment]
Thank you for the feedback and for merging.

**Pull request merged**